pywinauto/macos/ax_element_info.py

```diff
@@ -508,4 +509,6 @@ def control_id(self):
         # TODO: Discuss, is it a good idea to use hash as a control id?
         return -1
 
-
+    def get_avaliable_actions(self):
+        actions = get_list_of_actions(self)
+        print(actions)
```

Comment:
It's better to return list of strings. Calls `print` look like debug code.

Reply:
I have not yet been able to think of anything better than just returning a list of actions. For example how it is done in the get_list_of_actions/get_list_of_attributes method. Or I should do print/return how it is done in the print_control_identifiers method? Could you advise me, please?